src/TestRunnerOrchestrator.ts

```diff
@@ -168,7 +168,7 @@ export default class TestRunnerOrchestrator {
   }
 
   private createTestRunnerSandboxes(): Promise<TestRunnerSandbox[]> {
-    let cpuCount = os.cpus().length;
+    let cpuCount = this.options.cpuCount || os.cpus().length;
```

Comment:
@simondel  Agreed, sounds good.

@jeznag do you want to create a PR to add it to the [stryker-api](https://github.com/stryker-mutator/stryker-api)? Or shall we?